docs/index.adoc

```diff
@@ -9573,6 +9573,129 @@ Options with a <<Default Values,default value>> can use the `${DEFAULT-VALUE}` v
 userName=Specify the user name. The default is ${DEFAULT-VALUE}.
 ----
 
+=== Controlling the locale
+
+In your localized application, it may be desirable to specify the locale in order to determine the language of message texts and help output. One way of controlling the locale is to give `-Duser.language=de` as VM argument when running the app. A more accessible and user-friendly approach is to to implement a command line parameter (e. g. `--locale`) inside your app which can be used to change the locale. The latter technique requires a two-phase approach to parsing in your application in order to get a valid load order. The minimal example below demonstrates how to implement this two phase approach:
```

Comment:
Minor detail, but I like having one sentence per line.
(That makes it easier to read in editors that do not line-wrap automatically.)

Reply:
Changed as requested.